SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs

```diff
@@ -79,7 +77,11 @@ public IEnumerable<CareCaseData> GetProcessData(ListCasesRequest request)
                 throw new DocumentNotFoundException("document not found");
             }
 
-            var response = ResponseFactory.ToResponse(result);
+            var response = ResponseFactory
```

Comment:
For accurate limit and cursor values please set them after date filters have been applied